lib/auth/index.js

```diff
@@ -0,0 +1,50 @@
+import request from 'request';
+import passport from 'passport';
+import OAuth2Strategy from 'passport-oauth2';
+import { authenticateOrLogin, localUser } from './middleware';
+
+const {
+  GRAVITY_API_URL,
+  ARTSY_ID,
```

Comment:
Right, good catch, made the change. By the way @mzikherman, do remember which port you had opened up for testing on localhost? I think it was `8081` right?